staging/src/k8s.io/kubectl/pkg/cmd/util/editor/editoptions.go

```diff
@@ -312,7 +312,7 @@ func (o *EditOptions) Run() error {
 			if len(results.file) > 0 {
 				os.Remove(results.file)
 			}
-			klog.V(4).Infof("User edited:\n%s", string(edited))
+			klog.V(4).InfoS("User edited", string(edited))
```

Comment:
Is this a user-visible change? If it is, we should update the changelog entry from `NONE`.